src/server/worldserver/worldserver.conf.dist

```diff
@@ -3029,6 +3029,16 @@ NoGrayAggro.Above = 0
 NoGrayAggro.Below = 0
 
 #
+#    PreventRenameCharacterOnCustomization
+#        Description: if equal to 1 when players wants to do Customizations Can't Rename Character Too
+#                     work with all character customization commands.
+#        Default:     0 - Disable(character can rename at character Customizations)
+#                     1 - Enable(Can't Rename on Character Customizations)
+#
+
+PreventRenameCharacterOnCustomization = 0
```

Comment:
Please check if I have kept the original and intended meaning of the text. I have tried to follow the standards used in worldserver.conf.dist (simple and straight English text pertaining to the source):
```
#
#    PreventRenameCharacterOnCustomization
#        Description: If option is set to 1, player can not rename the character in character customization.
#                     Applies to all character customization commands.
#        Default:     0 - (Disabled, character can be renamed in Character Customization)
#                     1 - (Enabled, character can not be renamed in Character Customization)
#

PreventRenameCharacterOnCustomization = 0

#
```